otter/tap/api.py

```diff
@@ -286,6 +289,21 @@ def on_client_ready(_):
     return s
 
 
+def setup_converger(parent, kz_client, dispatcher):
+    """Create a Converger service."""
+    converger_buckets = range(1, 10)
+    partitioner_factory = partial(
+        Partitioner,
+        kz_client,
+        10,  # interval
+        '/converger_partition',
```

Comment:
Should this be `CONVERGENCE_PARTITIONER_PATH`?  That doesn't seem to be used elsewhere.